### ISSUE TITLE
Definition

### ISSUE THREAD

**Issue by synagonism** [aeternity/wiki#6]
wiki definition: "æternity is a blockchain protocol that is based on Ethereum’s network, ..."
my opinion:
"æternity is a blockchain network based on Ethereum's network protocol, ..."

more in my blockchain-network structured-concept page:
http://synagonism.net/dirMiwMcs/dirTchInf/filMcsBcnnet.html

**Comment by JasonJAyalaP** [comment]
I agree that "based on network" implies that it is build on top of Ethereum instead of inspired by. Where do you find this text?

**Comment by synagonism** [comment]
Was the first sentence at
https://github.com/aeternity/wiki/wiki/Home/16f3ec911c1ab5925a771df85bf22d87f5e8ff45
My point is other.
There are:
- æternity network
- æternity blockchain
- æternity protocol
I think 'æternity' by itself to mean 'æternity network'

**Comment by Zwilla** [comment]
I think æternity blockchain was / is written from scratch by ZACK (Erlang)
